test/jdk/jdk/internal/util/random/RandomSupportTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * @test
+ * @bug 8326227
+ * @modules java.base/jdk.internal.util.random
+ * @summary Verify that RandomSupport methods behave as specified
```

Comment:
Nit - can you move this `@summary` line after the `@bug` line? It would then match the recommended tag order https://openjdk.org/jtreg/tag-spec.html#ORDER

Reply:
Done.